### PR TITLE
UG: Add command format and logo

### LINKED ISSUE #114: UG: Add 'notes about command format'
Add in user guide as follows from AB3:
Screenshot 2021-03-26 at 11 44 28 AM

### LINKED ISSUE #117: UG: Update TutorBuddy Logo
Add our awesome logo to the top of our UG.

photo_2021-03-26 12 04 31

### PR DESCRIPTION
**Pull request by samleewy**
Resolve #114 - Add commond format 
Resolve #117 - Add TutorBuddy Logo

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -62,6 +64,30 @@ It allows for faster and more effective student management.
...
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the command format:**<br>
+
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+
+* Items in square brackets are optional.<br>
+  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```
Comment (enhao25) [inline docs/UserGuide.md]: Hmmmm actually using TAG in this case might not be intuitive as we do not have a tag parameter in our entire application. If we are to include this here, maybe a different parameter and including the edit_student in the UG would then be more useful.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -62,6 +64,30 @@ It allows for faster and more effective student management.
...
+
+* Items with `…`​ after them can be used multiple times including zero times.<br>
```
Comment (enhao25) [inline docs/UserGuide.md]: Using KEYWORDS during `find_student` as the example might be better than using tags here.

**Review by enhao25 — COMMENTED** [review]
LGTM, added some comments that we could do to improve on the UG to be more suited for our application.

File: docs/UserGuide.md (hunk 1 above)
Comment (samleewy) [reply docs/UserGuide.md]: Whoops was copied wholesale 😂 will make the change, thanks!

File: docs/UserGuide.md (hunk 2 above)
Comment (samleewy) [reply docs/UserGuide.md]: Ok thanks!!

File: docs/UserGuide.md (hunk 1 above)
Comment (samleewy) [reply docs/UserGuide.md:75]: Actually on second thought, let's just scrap this optional explanation until we have a command with optionals.

**Review by enhao25 — APPROVED** [review]
LGTM

**Pull request merged**